Sources/Utility/ArgumentParser.swift

```diff
@@ -382,12 +361,94 @@ public final class ArgumentParser {
         }
     }
 
+    /// An enum representing way to parse argument values.
+    public enum ParsingStrategy {
+        case one
+        case multiple
+        case remainder
```

Comment:
Sure, I'm okay with `collect`. `remainder` is the one used in `argparse` in case you want to keep the logical link.